test/Nancy.Tests/Unit/DynamicDictionaryValueFixture.cs

```diff
@@ -987,6 +987,38 @@ public void Should_return_default_value_if_implicit_convert_fails_on_datetime()
         }
 
         [Fact]
+        public void Should_adjust_to_universal_time_when_globalizationconfiguration_datetimestyles_requires_it()
+        {
+            // Given
+            var expected = DateTime.Parse("2016-05-24T08:41:37.1661565");
+
+            var config = new GlobalizationConfiguration(new [] {"en-US"}, dateTimeStyles: DateTimeStyles.AdjustToUniversal);
+            var value = new DynamicDictionaryValue("2016-05-24T10:41:37.1661565+02:00", config);

```

Comment:
Basically just want to have a test that makes sure it can adjust to universal and @khellang where worried that the test might break once daylight savings change because of the explicit `+2` offset